cmd/embedded-cluster/join.go

```diff
@@ -190,7 +190,7 @@ var joinCommand = &cli.Command{
 		}
 
 		setProxyEnv(jcmd.Proxy)
-		proxyOK, localIP, err := checkProxyConfigForLocalIP(jcmd.Proxy)
+		proxyOK, localIP, err := checkProxyConfigForLocalIP(jcmd.Proxy, "") // TODO (@salah): pass network interface
```

Comment:
still a todo here

Reply:
yeah, we decided to ship the feature for installs / restores and add it later for joins (multi nodes).